### ISSUE TITLE
Progressive enhancement for editors

### ISSUE THREAD

**Issue by Atulin** [Genfic/Ogma#27]
Requiring Javascript to even display something as simple and basic as the blog/story/chapter editors is not great. Perhaps for the tag selection and the like, but not for the title or the body.

Instead, progressive enhancement should be implemented, through adding functionality on top of a basic form. For example, the inputs with a progress bar can be replaced with just existing ASP inputs that utilize `asp-for` and the like, and a simple JS script that appends the progress bar whenever needed.
____

- [x] Blogpost editor
- [x] Chapter editor
- [x] Story editor
- [x] Folder editor
- [x] Club editor
- [x] Bookshelf editor

**Comment by Atulin** [comment]
First steps taken in https://github.com/Atulin/Ogma/commit/fa013a54e414b2e7ac815f5aaa0238333630c265, implement progressive enhancement for blogpost and chapter editors

**Comment by Atulin** [comment]
Club editor done with https://github.com/Atulin/Ogma/commit/515f81a0a07d7c9d8ae2837cfdd1fd2082a5b588

**Comment by Atulin** [comment]
Story editor done with https://github.com/Atulin/Ogma/commit/4a8f71d0fa6a3357b582ca32b72e53991a0661d8

**Comment by Atulin** [comment]
Regression for file inputs because of Fluent Validation: https://github.com/Atulin/Ogma/issues/56

**Comment by Atulin** [comment]
Regression fixed with https://github.com/Atulin/Ogma/commit/03c1feed8818a753e9817c151634209634d90f65

**Comment by Atulin** [comment]
Bookshelf editor doesn't need any improvements anymore, so issue can be closed
